docs/applications/bitcoin-core.md

```diff
@@ -18,26 +18,30 @@ To ensure that the official binary is used, we store the verified signing key `0
 
 ### Configuration
 
-The application configuration is specified in the local `/etc/bitcoin/bitcoin.conf` file. The `bitcoin.conf` file is written into the Armbian image by the  [`customize-armbian-rockpro64.sh`](../../armbian/base/build/customize-armbian-rockpro64.sh) script.
+The application configuration is specified in the local `/etc/bitcoin/bitcoin.conf` file. The `bitcoin.conf` file is created from a template into the Armbian image during the build process, and can be automatically updated during regular operations as well. These are the initial settings:
```

Comment:
```suggestion
The application configuration is specified in the local `/etc/bitcoin/bitcoin.conf` file. The `bitcoin.conf` file is generated from a template during the build process of the Armbian image. It can updated during regular operations. These are the initial settings:
```